packages/appkit/src/utils/TypesUtil.ts

```diff
@@ -95,4 +95,10 @@ export type AppKitOptions = {
    * @default "{ bip122: 'payment', eip155: 'smartAccount', polkadot: 'eoa', solana: 'eoa' }"
    */
   defaultAccountTypes?: Partial<OptionsControllerState['defaultAccountTypes']>
+
+  /**
+   * Enable or disable network switching in the modal.
+   * @default true
+   */
+  showNetworkSwitcher?: boolean
```

Comment:
I think we can better name this to `enableNetworkSwitch`. Cause if user switches from wallet it is also restricted and not related to network switcher